app/test/specs/requestInvitationControllerSpec.js

```diff
@@ -0,0 +1,123 @@
+'use strict';
+
+(describe('Test RequestInvitationController', function() {
+
+    var requestInvCtrl, httpBackend, scope, institutionService, createCtrl, state;
+
+    var SEARCH_INST_URI = "api/search/institution?";
+    var INST_URI = "/api/institutions/";
+
+    var splab = {
+        name: 'SPLAB',
+        key: '987654321',
+        id: '987654321',
+        photo_url: "photo_url"
+    };
+
+    var certbio = {
+        name: 'CERTBIO',
+        key: '123456789',
+        id: '123456789',
+        photo_url: "photo_url"
+    };
+
+    var maiana = {
+        name: 'Maiana',
+        key: '12107',
+        email: 'maiana.brito@ccc.ufcg.edu.br'
+    };
+
+    beforeEach(module('app'));
+
+    beforeEach(inject(function($controller, $httpBackend, $rootScope, $q, $state, InstitutionService) {
+        httpBackend = $httpBackend;
+        scope = $rootScope.$new();
+        state = $state;
+        institutionService = InstitutionService;
+
+        httpBackend.when('GET', 'institution/institution_page.html').respond(200);
+        httpBackend.when('GET', "main/main.html").respond(200);
+        httpBackend.when('GET', "home/home.html").respond(200);
+
+        createCtrl = function() {
+            return $controller('RequestInvitationController',
+                {
+                    scope: scope,
+                    institutionService: institutionService
+                });
+        };
+        requestInvCtrl = createCtrl();
+        httpBackend.flush();
+    }));
+
+    afterEach(function() {
+        httpBackend.verifyNoOutstandingExpectation();
+        httpBackend.verifyNoOutstandingRequest();
+    });
+
+    describe('RequestInvitationController functions', function() {
```

Comment:
o describe deve ser o nome da função a ser testada, e cada it ser a saída esperada pelo cenário.